src/frontend/src/lib/i18n/en.json

```diff
@@ -42,8 +42,8 @@
 	},
 	"auth": {
 		"text": {
-			"title_part_1": "A wallet that lives on-chain for",
-			"title_part_2": "100% decentralization",
+			"title_part_1": "Multi-chain wallet.",
+			"title_part_2": "100% onchain",
```

Comment:
On the screenshot that Andronikos shared in Slack, the second part also has a dot (`100% on chain.`).

Reply:
The foundation wants "onchain" I understand.

See for example this recent PR applied to the portal: https://github.com/dfinity/portal/pull/3530

Reply:
Oh true, it should be `100% onchain.`. The question about the dot is still pending though.

Reply:
> On the screenshot that Andronikos shared in Slack, the second part also has a dot (100% on chain.).

Good catch! Corrected